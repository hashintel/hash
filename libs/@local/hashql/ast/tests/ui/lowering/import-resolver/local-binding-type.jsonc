//@ run: pass
//@ description: Tests that local type bindings are properly tracked in scope and not resolved as paths
["type", "Dict", "Number", ["type", "Bar", "Dict", "_"]]
