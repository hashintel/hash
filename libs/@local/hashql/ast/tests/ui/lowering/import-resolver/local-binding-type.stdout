Expr#4294967040@27
  ExprKind (Type)
    TypeExpr#4294967040@27 (name: Dict)
      Type#4294967040@11
        TypeKind (Path)
          Path#4294967040@11 (rooted: true)
            PathSegment#4294967040@10 (name: kernel)
            PathSegment#4294967040@10 (name: type)
            PathSegment#4294967040@10 (name: Number)
      Expr#4294967040@26
        ExprKind (Type)
          TypeExpr#4294967040@26 (name: Bar)
            Type#4294967040@23
              TypeKind (Path)
                Path#4294967040@23 (rooted: false)
                  PathSegment#4294967040@22 (name: Dict)
            Expr#4294967040@25
              ExprKind (Underscore)
